homeassistant/components/integration/config_flow.py

```diff
@@ -87,6 +88,14 @@
                 translation_key=CONF_UNIT_TIME,
             ),
         ),
+        vol.Optional(CONF_MAX_AGE, default=0): selector.NumberSelector(
+            selector.NumberSelectorConfig(
```

Comment:
Would a time selector be better?

Reply:
Looked into this and I don't think it is better. It looks like that one is used to select a time of day, whereas we need a duration.

<img width="544" alt="Screenshot 2024-03-28 at 22 45 47" src="https://github.com/home-assistant/core/assets/15732230/61870889-d825-412e-b63c-25b4e7b0baf8">

Reply:
There's a duration selector you can use.
https://www.home-assistant.io/docs/blueprint/selectors/#duration-selector

Reply:
Thank you! Replaced the NumberSelector for the DurationSelector!